features/MAT001_Material-set.feature

```diff
@@ -0,0 +1,97 @@
+@implementer-agreement
+@MAT
+@version1
+@E00020
+Feature: MAT001 - Material Set
+  The rule verifies that correct names are used for material sets.
```

Comment:
```suggestion
  The rule verifies that correct names are used for material sets. In this rule, material sets are the implicit generalization of IfcMaterialConstituentSet, IfcMaterialLayerSet, IfcMaterialProfileSet.
```

Material Set is not a known thing in the schema, and we kind of haphazardly came up with this generalized concept for 4.3 when Dion and I were on a deadline. So I think it's helpful to clarify a bit what this means.